.github/workflows/package.yml

```
@@ -35,7 +35,7 @@
       run: mvn -B package --file pom.xml
       
     - name: Upload WAR file as artifact
-      uses: actions/upload-artifact@v2 
+      uses: actions/upload-artifact@v3 
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)